app/controllers/coordinator_controller.rb

```diff
@@ -22,6 +41,26 @@ def schedule
     end
   end
 
+  def match
+    respond_to do |format|
+      begin
+        ActiveRecord::Base.transaction do
+          Interest.destroy_all(donation_id: match_params[:donation_id])
+          Transaction.create(donation_id: match_params[:donation_id],
+                             recipient_id: match_params[:recipient_id])
+        end
+        format.json { render json: {}, status: :created }
+      rescue Exception => ex
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?<br>Useless assignment to variable - `ex`.